packages/odata-v4/src/request-builder/get-all-request-builder.spec.ts

```diff
@@ -155,5 +156,27 @@ describe('GetAllRequestBuilder', () => {
         .execute(defaultDestination);
       expect(actual).toEqual([createTestEntity(testEntity)]);
     });
+
+    it('parses the raw number of count response', async () => {
+      mockCountRequest(
+        defaultDestination,
+        4711,
+        TestEntity.requestBuilder().getAll()
+      );
+      const count = await requestBuilder.count().execute(defaultDestination);
+      expect(count).toBe(4711);
+    });
+  });
+
+  describe('execute raw', () => {
+    it('parses the raw number of count response', async () => {
```

Comment:
Why should this happen in the get all spec? ;)

Reply:
because count is part of the getAll? In v2 and common we have the same tests

Reply:
Hmm, not really. These tests check that `execute` and `executeRaw` of the `CountRequestBuilder` return what we expect them to return. The `GetAllRequestBuilder` only instantiates the `CountRequestBuilder`. So for a test in get all, I think we could check that a correct request builder is returned from the count function, but the response of the count request is not related to the get all request spec.